app/controllers/comments_controller.rb

```diff
@@ -107,6 +107,8 @@ def create
   rescue Pundit::NotAuthorizedError
     raise
```

Comment:
Unrelated to this PR, but why are we rescuing from this exception only to immediately [re-raise](https://www.exceptionalcreatures.com/guides/advanced-rescue-and-raise.html#reraising-exceptions) it?

> If you call `raise` with no arguments, while inside of a `rescue` block, Ruby will re-raise the original rescued exception.

😕

Reply:
I had the exact same question when this was created, [this is why](https://github.com/thepracticaldev/dev.to/pull/5485#discussion_r366056925).

Reply:
Thanks @mstruve, I'll literally make a PR to leave this as a comment in the code before we confuse more people with this in the future.